src/main/java/uk/ac/ebi/ena/app/utils/MenuUtils.java

```diff
@@ -221,4 +225,22 @@ private static boolean validateAccession(String firstColumn) {
         return false;
     }
 
+    public static boolean validateSearchRequest(String searchQuery) {
+        return (searchQuery.contains("portal/api/search") || searchQuery.contains("result="));
```

Comment:
must always have result. better to parse the query string into a map and check individual keys?